DEBUGGING.md

```diff
@@ -0,0 +1,51 @@
+# ember-m3
+
+## Debugging
+
+If you are used to interacting with `DS.Model` records, it can be intimidating to see
+the `m3` proxy object in the console:
+![Proxy object screenshot](screenshots/proxy.png)
+
+However, there are some convenience methods and tools to make it easier for you to understand
+exactly what data is inside the proxy.
+
+### Ember Inspector
+
+`m3` record types are supported by Ember Inspector. If you have the extension installed you can
+go the pane in devtools and click on the `Data` tab to see the records that are currently loaded into
+your application.
+![Ember Inspector screenshot](screenshots/ember_inspector.png)
+
+### debugJSON
+
+This method is used to get the JSON from a particular `m3` record in the store.
+It is fairly easy to use and is helpful if you already have a reference to the record and want to
+inspect all the attributes and values associated with it:
```

Comment:
Might be helpful to mention that this shows the data sent by the server, but does not include any unsaved local changes